package.json

```diff
@@ -48,6 +48,7 @@
     "semver": "7.3.2",
     "style-loader": "1.2.1",
     "styled-components": "4.1.1",
+    "use-count-up": "2.2.4",
```

Comment:
Worth a dependency addition?

Reply:
It's pretty small (24kb) and adds some motion to the dashboard.